internal/collector/pgbackrest.go

```diff
@@ -0,0 +1,105 @@
+// Copyright 2024 - 2025 Crunchy Data Solutions, Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package collector
+
+import (
+	"context"
+	_ "embed"
+	"encoding/json"
+	"fmt"
+	"slices"
+
+	"github.com/crunchydata/postgres-operator/internal/feature"
+	"github.com/crunchydata/postgres-operator/internal/naming"
+	"github.com/crunchydata/postgres-operator/pkg/apis/postgres-operator.crunchydata.com/v1beta1"
+)
+
+// The contents of "pgbackrest_logs_transforms.yaml" as JSON.
+//
+//go:embed "generated/pgbackrest_logs_transforms.json"
+var pgBackRestLogsTransforms json.RawMessage
+
+func NewConfigForPgBackrestRepoHostPod(
+	ctx context.Context,
+	repos []v1beta1.PGBackRestRepo,
+) *Config {
+	config := NewConfig()
+
+	if feature.Enabled(ctx, feature.OpenTelemetryLogs) {
```

Comment:
Note to self: everywhere else we do a `if !` and exit early, consider doing that